client/ayon_core/pipeline/publish/lib.py

```diff
@@ -717,6 +717,7 @@ def get_instance_staging_dir(instance):
     instance.data.update({
         "stagingDir": staging_dir_path,
         "stagingDir_persistent": staging_dir_info.persistent,
+        "stagingDir_custom": staging_dir_info.custom
```

Comment:
Why `_custom`? That does not indicate it should hold boolean, the same for the attribut, it should be `is_custom`.

Reply:
I did it personally because it matched the `persistent` one - and as described in the PR description I'd have personally preferred to store the `staging_dir_info` data class on the instance data instead.

Anyway, should we still refactor to `is_custom`? Both this core PR and the maya one aren't in a released version yet.